docroot/themes/humsci/humsci_basic/patterns/structured-card/structured-card.html.twig

```diff
@@ -1,12 +1,15 @@
 {%
   set classes = [
     'hb-card',
-    'hb-card--horizontal'
+    'hb-card--horizontal',
```

Comment:
At some point I think it might be better to not couple all these types of cards so tightly, but we can keep this for now.